packages/core/src/magicswap/magic-swap.test.ts

```diff
@@ -0,0 +1,130 @@
+import { arbitrumSepolia } from "viem/chains";
+import { describe, expect, it } from "vitest";
+
+import {
+  fetchPool,
+  fetchPools,
+  getAddLiquidityArgs,
+  getSwapArgs,
+  getSwapRoute,
+} from "./index";
+
+describe("magic-swap", () => {
+  // it("should fetch pairs", async () => {
+  //   const pools = await fetchPools({
+  //     chainId: arbitrumSepolia.id,
+  //     inventoryApiUrl: "https://trove-api-dev.treasure.lol",
+  //     inventoryApiKey: "oP5gtcE5Mb1vAoBz2J5Ad",
```

Comment:
Please remove the API keys in this file

Reply:
sorry, it wasn't my intention to commit this file, I'll remove it for now